backend/open_webui/storage/provider.py

```diff
@@ -206,7 +208,7 @@ def get_storage_provider(storage_provider: str):
         Storage = LocalStorageProvider()
     elif storage_provider == "s3":
         Storage = S3StorageProvider()
-    elif storage_provider == "gcs":
+    elif storage_provider == "gcs" and GCS_BUCKET_NAME:
```

Comment:
The error message above is very missleading if GCS_BUCKET_NAME is missing, I would just remove that check on GCS_BUCKET_NAME.